moviepy/audio/fx/audio_normalize.py

```diff
@@ -9,6 +9,7 @@ def audio_normalize(clip):
 
     Return an audio (or video) clip whose audio volume is normalized
     so that the maximum volume is at 0db, the maximum achievable volume.
+    Audio that is muted is left unchanged.
```

Comment:
I think that this string is not needed because that "muted audio is left unchanged" is the expected behaviour of normalization.

Reply:
Agreed

Reply:
Changed in aaf6300.